app/src/main/java/org/simple/clinic/forgotpin/createnewpin/ForgotPinCreateNewEffectHandler.kt

```diff
@@ -55,7 +55,7 @@ class ForgotPinCreateNewEffectHandler @AssistedInject constructor(
     return ObservableTransformer { effects ->
       effects
           .observeOn(schedulersProvider.io())
-          .flatMap { userSession.requireLoggedInUser() }
+          .map { userSession.loggedInUserImmediate() }
```

Comment:
We could also replace this by injecting the current user instead of using the `UserSession`.

Reply:
Ah yes, will do it in this PR.